spam/spam_tutorial.py

```diff
@@ -625,61 +597,127 @@ def plot_probabilities_histogram(Y_probs):
 X_test = vectorizer.transform(words_test)
 
 # %% [markdown]
-# * Filter out examples with no labels (always abstains)
+# ### Filtering Unlabeled Data Points
+
+# %% [markdown]
+# As we saw earlier, some of the data points in our training set received no weak labels from our LFs.
+# These examples are not helpful for training our classifier, as they convey no supervision signal, so we filter them out before training.
```

Comment:
do we expect them to hurt? 
maybe we can explain that we _tend to_ see better performance by filtering out the noisy/low confidence examples